app/services/foreman/plugin.rb

```diff
@@ -396,5 +405,30 @@ def extend_template_helpers_by_module(mod)
       end
       allowed_template_helpers(*(mod.public_instance_methods - Module.public_instance_methods))
     end
+
+    def add_manager_roles(resources, opts)
+      manager = Role.find_by :name => 'Manager'
+      add_permissions(manager, resources, opts) do |resource|
+        Permission.where(:resource_type => resource)
+      end
+    end
+
+    def add_viewer_roles(resources, opts)
+      viewer = Role.find_by :name => 'Viewer'
+      add_permissions(viewer, resources, opts) do |resource|
+        Permission.where(:resource_type => resource).where(Permission.arel_table[:name].matches("view_%"))
```

Comment:
just out of curiosity, why using `arel_table` here? do you prefer this over the string in condition?